samples/hyperparameter-tutorial/src/steps/01_prep_data.py

```diff
@@ -0,0 +1,37 @@
+from azureml.core.run import Run, _OfflineRun
+from azureml.core import Workspace
+from sklearn.datasets import load_diabetes
+import numpy as np
+import argparse
+import os
+
+# Get context
```

Comment:
No, these are needed.
There are used to handle arguments and directory.

Reply:
Still can't find any references in the prep_data script to `run.` or `ws.`. Doesn't argparse handle the arguments?

Reply:
Oh! I'm sorry. I misunderstood it. As you said, _run_ and _ws_ are not used.
However, I think it would be better to be left here because _run_ and _ws_ could be kind of pattern to re-use context within each script of steps.